packages/core/src/test/notifications/rendering.test.ts

```diff
@@ -0,0 +1,235 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as sinon from 'sinon'
+import assert from 'assert'
+import { ToolkitNotification } from '../../notifications/types'
+import fs from '../../shared/fs/fs'
+import path from 'path'
+import { tempDirPath } from '../../shared/filesystemUtilities'
+import { getTestWindow } from '../shared/vscode/window'
+import { panelNode } from './controller.test'
+
+describe('Notifications Rendering', function () {
```

Comment:
This suite is using a lot of stubs... why? We should have methods to detect opened messages and text documents.